go.mod

```diff
@@ -152,3 +152,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace go.mongodb.org/atlas-sdk/v20230201002 => github.com/mongodb/atlas-sdk-go/v20230201002 v20230201002.0.1-0.20230713083610-8d5d78c43b51
```

Comment:
Why are you using replace here? Can't you use `go get go.mongodb.org/atlas-sdk/v20230201002@latest` or `go get go.mongodb.org/atlas-sdk/v20230201002@v20230201002.0.1-0.20230713083610-8d5d78c43b51`?

Reply:
So I wanted specifically to make use of `dev-latest` here, since the Streams API isn't currently exposed.

When I tried using dev-latest, at least how I was getting it meant I needed this `replace` syntax for it to work.